gensrc/proto/lake_types.proto

```diff
@@ -35,8 +34,10 @@ message DelvecCacheKeyPB {
 }
 
 message DelvecMetadataPB {
+    // Delvec file name
+    optional string delvec_file = 1;
```

Comment:
Should save the file name in `DelvecCacheKeyPB`

Reply:
Instead of saving the file name string, maybe we can save the binary UUID (`PUniqueId`) to reduce the structure size.